deku-p/src/core/bin/node/deku_node.ml

```diff
@@ -118,9 +118,9 @@ let main params style_renderer log_level =
     if port = 4440 then api_uri :: validator_uris else validator_uris
   in
 
-  (* The VM must be started before the node because this call is blocking
-     Logs.info (fun m ->
-         m "Starting IPC with external vm at path %s" named_pipe_path); *)
+  (* The VM must be started before the node because this call is blocking 
```

Comment:
Be sure to run `dune build @fmt --auto-promote`.

Reply:
Done.